Formula/liquigraph.rb

```diff
@@ -0,0 +1,23 @@
+class Liquigraph < Formula
+  desc "Migration runner for Neo4j"
+  homepage "http://www.liquigraph.org"
+  url "https://github.com/fbiville/liquigraph/archive/liquigraph-1.0-RC3.tar.gz"
+  sha256 "9b616bd8d8798374d316decd7c1ef60882bf81f6627511af4e3c710143882c47"
+  head "https://github.com/fbiville/liquigraph.git"
+
+  depends_on "maven" => :build
+  depends_on :java => "1.7+"
+
+  def install
+    system "mvn", "-q", "clean", "package", "-DskipTests"
+    mkdir "binaries"
+    system "tar", "xzf", "liquigraph-cli/target/liquigraph-cli-bin.tar.gz", "-C", "binaries"
+    mv "binaries/liquigraph-cli/liquigraph.sh", "binaries/liquigraph-cli/liquigraph"
```

Comment:
Rather than the `mv` step here and the one below it (`.install` is more or less `mv`) you could do something like:

```ruby
libexec.install "binaries/liquigraph-cli/liquigraph.sh" => "liquigraph"
libexec.install "binaries/liquigraph-cli/liquigraph-cli.jar"
```
That handles the rename for you as well.